### PR TITLE
fixes gh-1685 Split box topics for version 2.6

### PR DESCRIPTION
**Pull request by Onvember**
Split + translate + replace :ref: with :doc: for:

- [x] Submodule box.backup
- [x] Submodule box.cfg
- [x] Submodule box.ctl
- [x] Submodule box.error
- [x] Submodule box.index
- [x] Submodule box.info
- [x] Function box.once
- [x] Constant box.NULL
- [x] Submodule box.schema
- [x] Sequences
- [x] Submodule box.session
- [x] Submodule box.slab
- [x] Submodule box.stat
- [x] Function box.snapshot
- [x] Functions for transaction management
- [x] Submodule box.space
- [x] Submodule box.tuple
- [x] Functions for SQL

### CODE REVIEW

File: doc/reference/reference_lua/box_index/unique.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.. _box_index-unique:
+
+===============================================================================
+index_object:unique
```
Comment (NickVolynkin) [inline doc/reference/reference_lua/box_index/unique.rst:4]: This should have a dot: `index_object.unique`

File: doc/reference/reference_lua/box_index/type.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.. _box_index-type:
+
+===============================================================================
+index_object:type
```
Comment (NickVolynkin) [inline doc/reference/reference_lua/box_index/type.rst:4]: This should have a dot: `index_object.type`

File: doc/reference/reference_lua/box_space/id.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.. _box_space-id:
+
+===============================================================================
+space_object:id
```
Comment (NickVolynkin) [inline doc/reference/reference_lua/box_space/id.rst:4]: This should have a dot: `space_object.id`

File: doc/reference/reference_lua/box_info/election.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      - state: follower
+        vote: 0
+        leader: 0
+        term: 1
+      ...
+
+   IDs in the ``box.info.election`` output are the replica IDs visible in
+   the ``box.info.id`` output on each node and in the ``_cluster`` space.
+
+   :ref:`State <repl_leader_elect_state>` can be ``leader``, ``follower``,
```
Comment (NickVolynkin) [inline doc/reference/reference_lua/box_info/election.rst:31]: WARNING: undefined label: repl_leader_elect_state (if the link has no caption the label must precede a section header)

**Review by NickVolynkin — APPROVED** [review]

File: doc/reference/reference_lua/box_index/unique.rst (hunk 1 above)
Comment (Onvember) [reply doc/reference/reference_lua/box_index/unique.rst]: fixed in 22a984c

File: doc/reference/reference_lua/box_index/type.rst (hunk 2 above)
Comment (Onvember) [reply doc/reference/reference_lua/box_index/type.rst]: fixed in 22a984c

File: doc/reference/reference_lua/box_space/id.rst (hunk 3 above)
Comment (Onvember) [reply doc/reference/reference_lua/box_space/id.rst]: fixed in 22a984c

File: doc/reference/reference_lua/box_info/election.rst (hunk 4 above)
Comment (Onvember) [reply doc/reference/reference_lua/box_info/election.rst:31]: will be fixed later in https://github.com/tarantool/doc/issues/1773

**Pull request merged**